src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -444,7 +444,7 @@
     <Compile Include="$(CommonPath)\Microsoft\Data\ProviderBase\DbConnectionPoolGroup.cs">
       <Link>Common\Microsoft\Data\ProviderBase\DbConnectionPoolGroup.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Microsoft\Data\ProviderBase\TimeoutTimer.cs">
+    <Compile Include="..\..\src\Microsoft\Data\ProviderBase\TimeoutTimer.cs">
```

Comment:
Nit: Let's move this block to Line 322 with other shared files, when you see 
`  <!-- Contains common items shared between NetFx and NetCore -->`

Reply:
Thanks, I'll move it to line 79 where the rest of the `ProviderBase` compile link in shared items for NetFx and NetCore goes.